### PR TITLE
Added UploadToFileShare

### PR DESCRIPTION
**Pull request by fb0d3nheimer**
As discussed between Ralf and Olaf.

### CODE REVIEW

File: vocabularies/PDF.xml [hunk 1]
```diff
@@ -115,6 +115,12 @@
             <String>PDF supports setting the text direction-layout (e.g. left-to-right or right-to-left) in the document description</String>
           </Annotation>
         </Property>
+        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
+          <Annotation Term="Core.LongDescription">
+            <String>Is the upload of a PDF document to a file share server supported?</String>
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:121]: Why "to a file share server"? Does the client need to know that?

Comment (fb0d3nheimer) [reply vocabularies/PDF.xml]: Yes, the upload to a "File Share" is the specific feature that is advertised here. See also the adapted 
LongDescription.

File: vocabularies/PDF.xml [hunk 2]
```diff
@@ -115,6 +115,15 @@
             <String>PDF supports setting the text direction-layout (e.g. left-to-right or right-to-left) in the document description</String>
           </Annotation>
         </Property>
+        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
+          <Annotation Term="Core.LongDescription">
+            <String>Is the upload of a PDF document to a file share server supported?
+A file share connection needs to be configured on the server. 
+And the response of a corresponding request is then a "301 Moved Permanently" with the link to the document on the file share server. 
+            </String>
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:124]:
```suggestion
            <String>A file share connection needs to be configured on the server. 
The response of a corresponding request is then "301 Moved Permanently" with the link to the document on the file share server. 
            </String>
```
Is the link in a `Location` header, or how is it transported?

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: vocabularies/PDF.xml [hunk 3]
```diff
@@ -115,6 +115,14 @@
             <String>PDF supports setting the text direction-layout (e.g. left-to-right or right-to-left) in the document description</String>
           </Annotation>
         </Property>
+        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
+          <Annotation Term="Core.LongDescription">
+            <String>A file share connection needs to be configured on the server. 
+The response of a corresponding request is then `301 Moved Permanently` with a `Location` header containing the link to the document on the file share server. 
+            </String>
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:123]:
```suggestion
The response of a corresponding request is then `301 Moved Permanently` with a `Location` header containing the link to the document on the file share server.</String>
```

**Review by ralfhandl — DISMISSED** [review]

File: vocabularies/PDF.xml [hunk 4]
```diff
@@ -115,6 +115,13 @@
             <String>PDF supports setting the text direction-layout (e.g. left-to-right or right-to-left) in the document description</String>
           </Annotation>
         </Property>
+        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:119]:
```suggestion
        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
          <Annotation Term="Common.Experimental" />
          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
```

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**